build-a-test-app.md

```diff
@@ -9,6 +9,33 @@ toc: false
 
 ## Python
 
+~~~ python
+# Import the driver.
+import psycopg2
+
+# Assuming a "bank" database already exists, connect to it.
+conn = psycopg2.connect(database='bank', user='root', host='ROACHs-MacBook-Pro.local', port=26257)
+
+# Open a cursor to perform database operations.
+cur = conn.cursor()
```

Comment:
It's not entirely clear from the docs, but I don't think you're supposed to reuse cursor objects for multiple statements (it works in most databases, but some have problems with it). Of course, it works fine for postgres and cockroach so I don't think it's really worthwhile to do everything the 100% correct way for this example. But this is an example of how the low-level python database driver interface is a pain to work with, and people will generally use a layer on top of it. 

I think it's good that we have an example here that uses nothing but `psycopg2` (and doing it in one big code block is fine), but later on when we expand the scope of our examples we should probably be showing off the use of something like sqlalchemy or sqlobject (assuming we work with them; I haven't tried recently but we didn't work with sqlalchemy last time I looked).